src/main/java/io/supertokens/config/Config.java

```diff
@@ -57,9 +58,16 @@ private Config(Main main, String configFilePath) throws InvalidConfigException,
     private Config(Main main, JsonObject jsonConfig) throws IOException, InvalidConfigException {
         this.main = main;
         final ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
-        CoreConfig config = mapper.readValue(jsonConfig.toString(), CoreConfig.class);
-        config.normalizeAndValidate(main);
-        this.core = config;
+        try {
```

Comment:
this is for when loading from file right? Or for a tenant as well? Are the error messages for loading config from file consistent with that when adding / modifying config for a tenant?

Reply:
What happens if the values are wrong in the config.yaml file? What is the output of the core startup look like there?

Reply:
Updated to show same error message. it would look like this:

```Tenant(, public, public) | What caused the crash: io.supertokens.pluginInterface.exceptions.InvalidConfigException: 'access_token_validity' must be of type long```